base/operators.jl

```diff
@@ -1063,7 +1063,8 @@ splat(f) = args->f(args...)
     in(x)
 
 Create a function that checks whether its argument is [`in`](@ref) `x`, i.e.
-a function equivalent to `y -> y in x`.
+a function equivalent to `y -> y in x`. See also [`insorted`](@ref) for the use
+in sorted collections.
```

Comment:
```suggestion
with sorted collections.
```
?

Reply:
One last comment, maybe a native speaker could quickly help, and then we merge this.